### DISCUSSION TITLE
Datasheet for Ryzen AI IPUs?

### DISCUSSION THREAD

**Discussion by kele14x** [AMDResearch/Riallto#11] [Q&A] [answered]
Hi,

Where could I find the datasheet or detail information about the IPU inside Ryzen AI CPU? For example how many tiles and the size of row and column. Does the PE inside IPU is AIE or AIE-ML just the same as Versal, or was it something new?

For Versal devices, Xilinx has great documents list the details in the datasheet:

![image](https://github.com/AMDResearch/Riallto/assets/4020067/91920730-50c7-4e07-ae7e-c19bb2557fa6)

Was there any document for Ryzen IPU?

Thanks.

**Comment by mariodruiz** [comment, marked as answer]
Hi @kele14x,

You can find the information you are looking for in the Riallto material. Check out notebooks in section 3: https://riallto.ai/1_0_Introduction.html

**Reply by kele14x** [reply]
Got it, thanks!
